nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -358,31 +417,114 @@ def permuted_ols(
     verbose : int, optional
         verbosity level (0 means no message). Default=0.
 
+    masker : :obj:`~nilearn.maskers.NiftiMasker` or \
+            :obj:`~nilearn.maskers.MultiNiftiMasker`, optional
+        Mask to be used on data. This is necessary for TFCE-based inference.
+
+        .. versionadded:: 0.9.2
+
+    tfce : :obj:`bool`, optional
+        Whether to calculate :term:`TFCE` as part of the permutation procedure
+        or not.
+        Calculating TFCE values in each permutation can be time-consuming,
+        so this option is disabled by default.
+        The TFCE calculation is implemented as described in
+        :footcite:t:`smith2009threshold`.
+        Default=False.
+
+        .. versionadded:: 0.9.2
+
+    output_type : {'legacy', 'dict'}, optional
```

Comment:
Same comment on deprecated option `legacy` as in my review of #3181

Reply:
I think this should be fixed in #3181 now. Is this good to merge then?